wizard/src/utils/detect_location.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DetectedDetails, ConfidenceLevel } from "../types";
+
+enum LocationType {
+  // None indicates a non-Location Type.
+  None = "None",
+  LatLon = "LatLon",
+  Country = "Country",
+  State = "State",
+  Province = "Province",
+  Municipality = "Municipality",
+  County = "County",
+  City = "City",
+}
+
+enum CountryFormat {
```

Comment:
Should we call it / map it to a property?

Reply:
done